### ISSUE TITLE
Crash when using knives

### ISSUE THREAD

**Issue by Fred09x** [SoraZodia/Cannibalism#25]
Hello, I am a bit new to mods so I am not very sure why my server crashes when using the knives that the mod gives, that is, when I use them on myself it does not give a problem but when I use them Against another person the game crashes and I don't know what to do to solve the problem, I would appreciate any help, thank you.

**Comment by SoraZodia** [comment]
Could you provide a crash log for me to look at? And is there any other mods installed?

**Comment by cmoro-deusto** [comment]
It's a bug. I recon you cannot do anything untill a fix is released.

@SoraZodia the crash is at JSONConfig.checkEntity. A null check is missing, should be easy to fix. Let me know if you need help with this, I could try to greate a PR.

```mc_1  | [17-10-2021 22:29:36] [Server thread/FATAL]: Error executing task
mc_1  | java.util.concurrent.ExecutionException: java.lang.NullPointerException
mc_1  |         at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_212]
mc_1  |         at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_212]
mc_1  |         at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
mc_1  |         at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:723) [MinecraftServer.class:?]
mc_1  |         at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
mc_1  |         at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
mc_1  |         at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
mc_1  |         at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
mc_1  | Caused by: java.lang.NullPointerException
mc_1  |         at sorazodia.cannibalism.config.JSONConfig.checkEntity(JSONConfig.java:199) ~[JSONConfig.class:?]
mc_1  |         at sorazodia.cannibalism.items.ItemKnife.func_111207_a(ItemKnife.java:157) ~[ItemKnife.class:?]
mc_1  |         at net.minecraft.item.ItemStack.func_111282_a(ItemStack.java:391) ~[aip.class:?]
mc_1  |         at net.minecraft.entity.player.EntityPlayer.func_190775_a(EntityPlayer.java:1220) ~[aed.class:?]
mc_1  |         at net.minecraft.network.NetHandlerPlayServer.func_147340_a(NetHandlerPlayServer.java:1075) ~[pa.class:?]
mc_1  |         at net.minecraft.network.play.client.CPacketUseEntity.func_148833_a(SourceFile:69) ~[li.class:?]
mc_1  |         at net.minecraft.network.play.client.CPacketUseEntity.func_148833_a(SourceFile:13) ~[li.class:?]
mc_1  |         at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
mc_1  |         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_212]
mc_1  |         at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_212]
mc_1  |         at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]```

**Comment by SoraZodia** [comment]
Should be fixed now in v5.6.8, let me know if it's still an issue.
